dashboard/app/controllers/level_starter_assets_controller.rb

```diff
@@ -0,0 +1,69 @@
+class LevelStarterAssetsController < ApplicationController
+  S3_BUCKET = 'cdo-v3-assets'.freeze
+  S3_PREFIX = 'starter_assets/'.freeze
+
+  # GET /level_starter_assets/:id
+  def show
+    level_channel_id = params[:id]
```

Comment:
within dashboard, I'd suggest `level_starter_assets_id` as the name for this kind of id. if that's too long, you could do `starter_assets_id`.